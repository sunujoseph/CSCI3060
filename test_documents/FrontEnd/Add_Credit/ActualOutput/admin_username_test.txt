Enter Username: 
Enter User Name To Add Credit To: 
Error: No User With That Name Exists
Enter User Name To Add Credit To: 