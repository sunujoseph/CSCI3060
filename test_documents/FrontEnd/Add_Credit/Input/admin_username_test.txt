login
admin1
addcredit
nonusername
