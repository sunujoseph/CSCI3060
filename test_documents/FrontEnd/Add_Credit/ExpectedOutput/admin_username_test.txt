Enter Username:
Enter User Name To Add Credit To:
Cannot add credit to user than does not exist.